packages/fetch-and-compile/test/fixture.js

```diff
@@ -0,0 +1,64 @@
+        "ContractName": "GrowthVault",
+        "CompilerVersion": "v0.8.9+commit.e5eed63a",
+        "OptimizationUsed": "1",
+        "Runs": "200",
+        "ConstructorArguments": "000000000000000000000000000000000000000000000000000000000000014000000000000000000000000000000000000000000000000000000000000001800000000000000000000000003324af8417844e70b81555a6d1568d78f4d4bf1f000000000000000000000000000000000000000000000000000000000000000a00000000000000000000000000000000000000000000000000000000000000010000000000000000000000001948abc5400aa1d72223882958da3bec643fb4e50000000000000000000000001b02da8cb0d097eb8d57a175b88c7d8b47997506000000000000000000000000c2132d05d31c914a87c6611c10748aeb04b58e8f000000000000000000000000d3b71117e6c1558c1553305b44988cd944e97300000000000000000000000000a5e0829caced8ffdd4de3c43696c57f7d7a678ff0000000000000000000000000000000000000000000000000000000000000007536861726520560000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000025356000000000000000000000000000000000000000000000000000000000000",
+        "EVMVersion": "Default",
+        "Library": "",
+        "LicenseType": "MIT",
+        "Proxy": "0",
+        "Implementation": "",
+        "SwarmSource": "ipfs://4b803eff2b53da590615ae9571dfcb7a4fa564c5f1d196b7e24bac3dcf52f96d"
+      }
+    ]
+  }
+  module.exports = {mainnetData,arbitrumData,polygonData}
```

Comment:
Sorry for more of this nipitcking, but... there should be spaces here. `{ mainnetData, arbitrrumData, polygonData }`.